src/test/java/com/indeed/jiraactions/jiraissues/TestJiraIssuesProcess.java

```diff
@@ -122,14 +124,16 @@ public void testDateFilter() {
 
         final String[] issue2 = {"B", "Open", "0", "0", "20180801"};     // last updated 2018-08-01
         final Map<String, String> output2 = process.compareAndUpdate(issue2);
-        final String[] expected2 = {"B", "Open", "1546322400", "86400", "20180801"};
+        final String[] expected2 = {"B", "Open", unixtime, "86400", "20180801"};
         Assert.assertEquals(expected2, output2.values().toArray());
     }
 
     @Test
     public void testDaylightSavings() {
-        final DateTime date1 = DateTime.parse("2018-03-12");    // Daylight savings for 2018 begins: March, 11
-        final DateTime date2 = DateTime.parse("2018-11-05");    // Daylight savings for 2018 ends: November, 4
+        final DateTime date1 = JiraActionsUtil.parseDateTime("2018-03-12 00:00:00");    // Daylight savings for 2018 begins: March, 11
```

Comment:
DateTime.parse() changes the timezone to -5 when daylight savings occurs. The date replicates the start date, which will always be GMT-6, so we want to use JiraActionsUtil.parseDateTime() here instead to always keep it at GMT-6.